config/default.yml

```diff
@@ -1497,3 +1497,9 @@ Rails/SkipsModelValidations:
 Rails/Validation:
   Include:
     - app/models/**/*.rb
+
+Bundler/OrderedGems:
+  EnforcedStyle: no_comment_lines_split
+  SupportedStyles:
+    - no_comment_lines_split
```

Comment:
I'm not sure this has to be a supported style - probably we really need an option named "TreatCommentsAsGroupSeparators".